src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -593,18 +593,24 @@ OperationalCertStatus ConvertToNOCResponseStatus(CHIP_ERROR err)
     {
         return OperationalCertStatus::kInvalidPublicKey;
     }
-    if (err == CHIP_ERROR_INVALID_FABRIC_INDEX || err == CHIP_ERROR_WRONG_NODE_ID)
+    if (err == CHIP_ERROR_WRONG_NODE_ID || err == CHIP_ERROR_WRONG_CERT_DN || err == CHIP_ERROR_FABRIC_MISMATCH_ON_ICA)
     {
         return OperationalCertStatus::kInvalidNodeOpId;
```

Comment:
No, I don't think this is right...

For example, the "fabric mismatch on ICA" error should be InvalidNOC per spec, no?  InvalidNodeOpId is specifically for

> The Node Operational ID in the NOC is not formatted correctly.

or

> if the matter-node-id attribute in the subject DN of the NOC has a value outside the Operational Node ID range

in the prose.  But either way the fabric id not matching is not InvalidNodeOpId.

And WRONG_CERT_DN could mean a bunch of things too, right?

And CHIP_ERROR_WRONG_NODE_ID is not returned by _anything_ in the fabric table or certificate validation code...

Ideally what we would have are CHIP_ERROR values that correspond to the actual values of this enum in the spec, use them in `DeviceCommissioner::ConvertFromOperationalCertStatus`, and here maybe check for some additional possible errors but generally aim to have the "right" errors returned by things that know they are dealing with the opcerts cluster.